feature_engine/encoding/count_frequency.py

```diff
@@ -78,6 +78,10 @@ class CountFrequencyEncoder(CategoricalInitExpandedMixin, CategoricalMethodsMixi
 
     {errors}
 
+    threshold: int,float or None, default=None
+        With threshold you have the option to perform backoff binning for all
```

Comment:
backoff binning is how the author calls this in the book, right? 

I don't think people will understand this. We need to be more specific of what threshold is doing, for the lay audience. 

Something like: if a category is present in less than threshold number / fraction of observations, it will be grouped with other labels, and then replaced by the number/fraction of observations seen in all this rare labels combined. If encoding_method is 'count' then threshold takes an integer. If encoding_method is frequency, then threshold takes a float. If None, no grouping will be performed

I would also add a See Also section in the docstrings pointing to the rarelabelencoder for reference:

```
See Also
---------
feature_engine.encoding.RareLabelEncoder
```